package.json

```diff
@@ -16,25 +17,59 @@
     "@types/node": "^16.11.43",
     "@types/react": "^18.0.15",
     "@types/react-dom": "^18.0.6",
+    "@types/react-resizable": "^3.0.4",
+    "@types/react-router-dom": "^5.1.5",
     "@types/react-syntax-highlighter": "^15.5.5",
+    "@types/react-virtualized": "^9.21.10",
+    "@types/styled-components": "^5.1.9",
+    "@types/uuid": "^8.0.0",
+    "apexcharts": "^3.37.1",
+    "copy-to-clipboard": "^3.3.1",
     "d3-state-visualizer": "^1.6.0",
     "dagre": "^0.8.5",
     "github-markdown-css": "^5.1.0",
+    "highlightjs": "^9.16.2",
     "html-to-image": "^1.11.4",
+    "i18next": "^19.5.2",
+    "idb": "^5.0.8",
+    "json-server": "^0.17.2",
```

Comment:
This is probably devDependencies, right?

Reply:
Json server for sure yes and I think the others are used by the Timeline component from Metaflow (so à priori not dev)